lib/src/flutterflow_api_client.dart

```diff
@@ -64,6 +68,7 @@ Future<String?> exportCode({
   required bool exportAsModule,
   bool format = true,
   String? branchName,
+  bool exportAsDebug = false,
```

Comment:
Let's add a check that `exportAsModule` and `exportAsDebug` aren't both `true` at the same time.